stl/inc/span

```diff
@@ -0,0 +1,758 @@
+// span standard header

```

Comment:
`__msvc_all_public_headers.hpp` also needs to be updated.

Reply:
Done